Maven.gitignore

```diff
@@ -7,3 +7,6 @@ release.properties
 dependency-reduced-pom.xml
 buildNumber.properties
 .mvn/timing.properties
+
+# Exclude maven wrapper
+!/.mvn/wrapper/maven-wrapper.jar
```

Comment:
I'll add my 2 cents too :-)

> Is it common for people to use Java.gitignore without Maven.gitignore?

I would say it is very common to use Java.gitignore without Maven.gitignore. 
Maven is just one of many Java Build-Tools (there are also Gradle [which became more and more popular during the last years], SBT [mostly for Scala], Ant, Kobalt and probably more). Also you don't have to use a build-system at all. You can also use the Java-Compiler (javac) directly (even if this is not common). 

Java.gitignore also is not just for "real" Java project, but actually for the Java Platform. You would use it for all projects, that are based on the JVM - also Scala, Groovy, Kotlin, ... projects. 
And then you would add more gitignore (for build-tools, languages, transpilers), depending on your project setup.